### PR TITLE
Drop `display` field

### PR DESCRIPTION
**Pull request by yinm**
resolve: https://github.com/tsconfig/bases/issues/180

### CODE REVIEW

**Review by orta — CHANGES_REQUESTED** [review]
I'm open to this being post-processed out when we are going from the file in this repo to the file in the npm package, but not for removing the display tag inside these json files.

Having _display in ensures that each base json file is fully self contained

**Comment by yinm** [comment]
Thank you for your review.

I changed code that drop `display` field when generating npm package.
Most of the codes are unusable, so I removed all commits and rebase latest main branch.

Does it accord that you think？

File: bases/recommended.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es2016",
+    "target": "ES2015",
```
Comment (orta) [inline bases/recommended.json:3]:
```suggestion
    "target": "ES2016",
```

**Comment by orta** [comment]
Seems reasonable to me 👍🏻

**Review by orta — APPROVED** [review]

File: bases/recommended.json (hunk 1 above)
Comment (yinm) [reply bases/recommended.json]: Fix 7ca1d5c...15ede8e

**Comment by orta** [comment]
Thanks!

**Pull request merged**